src/components/editor/contentEditor.ts

```diff
@@ -14,35 +15,28 @@ type ProsemirrorModule = Awaited<typeof prosemirrorPromise>;
 let prosemirrorModule: ProsemirrorModule | undefined;
 prosemirrorPromise.then(pm => prosemirrorModule = pm);
 
-/**
- * Note: Cell implements this interface.
- */
-export interface ContentContainer {
-    textContent: string;
-}
-
 /**
  * The main WYSIWYM (what you see is what you mean) content editor for Markdown content in Starboard.
  */
 @customElement('starboard-content-editor')
 export class StarboardContentEditor extends LitElement {
     view?: EditorView<any>;
 
-    @property({type: Object})
-    content: ContentContainer;
+    private cell: Cell;
+    private runtime: Runtime;
 
     createRenderRoot() {
         return this;
     }
 
-    constructor(content: ContentContainer = {textContent: ""}, opts: {focusAfterInit?: boolean} = {}) {
```

Comment:
The reason I abstracted it to this type is because it makes it more portable (it may also make sense to do the same for the code editors later on, I wish I did it from the start). Smaller interfaces makes for more portable code.

So we could define some sort of interface of stuff we need to make something *editable*, like this:
```typescript
// We can rename this to something better: `TextContentEditable`?
interface ContentContainer {
  textContent: string;
}
```
The current `Cell` object would satisfy this interface, but importantly we can now also re-use it as a text editor for something else! So let's say at some point we want a text editor for the metadata of a cell we could use it for that too, or if I'm making a plugin that embeds multiple markdown editors I could reuse this component for that.

To listen for assignments to the `textContent` field we use a [ES6 proxy](https://github.com/gzuidhof/starboard-notebook/blob/master/src/components/helpers/cellProxy.ts#L44-L58). That way we can act on changes from the text editor if we need to.

To solidify the relationship we could put `implements ContentContainer` on our `Cell` type.

Long story short: let's keep the arg here as `ContentContainer` (or a better name!).